ublox/src/ubx_packets/packets.rs

```diff
@@ -403,7 +403,7 @@ impl NavPvtFlags3 {
         self.invalid_llh
     }
 
-    /// F9R interface descritpion document specifies that this byte is unused
+    /// F9R interface descriptions document specifies that this byte is unused
```

Comment:
```suggestion
    /// F9R interface description document specifies that this byte is unused
```

Reply:
Sorry @gwbres , I missed that. Good catch.

Reply:
same error on line 418